src/main/java/com/conveyal/gtfs/loader/JdbcTableWriter.java

```diff
@@ -180,11 +180,11 @@ public String update(Integer id, String json, boolean autoCommit) throws SQLExce
                     // If a referencing table has the current table as its parent, update child elements.
                     JsonNode childEntities = jsonObject.get(referencingTable.name);
                     if ((referencingTable.name.equals(Table.PATTERN_LOCATION.name) ||
-                        referencingTable.name.equals(Table.PATTERN_LOCATION_GROUP.name)) &&
+                        referencingTable.name.equals(Table.PATTERN_STOP_AREA.name)) &&
                         (hasNoChildEntities(childEntities))
                     ) {
                         // This is a backwards hack to prevent the addition of pattern location breaking existing
-                        // pattern functionality. If pattern location or pattern location group is not provided set to
+                        // pattern functionality. If pattern location or pattern stop are is not provided set to
```

Comment:
Typo!

Reply:
Thanks. Sorted.

Reply:
I can hear the woman saying See it Say it Sorted